src/components/HomePage/HomepageOncallCard.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023 Cortex Applications, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {useApi, identityApiRef} from "@backstage/core-plugin-api";
+import {cortexApiRef} from "../../api";
+import {useAsync} from "react-use";
+import {EmptyState, InfoCard, Progress, WarningPanel} from "@backstage/core-components";
+import {isNil} from "lodash";
+import moment from "moment";
+import {Oncall} from "../../api/types";
+import {Typography} from "@material-ui/core";
+import {getDisplayTimeUntilNextOncall, getOncallSchedulesDisplayName} from "./HomePageUtils";
+import {useHomepageOncallStyles} from "../../styles/styles";
+
+export const getDisplayForCurrentOncalls = (oncalls: Oncall[], classes: any) => {
+    return (
+    <InfoCard
+      title="On-Call"
+      className={classes.root}
+    >
+        <Typography>
+            You're currently on call for {getOncallSchedulesDisplayName(oncalls)}
+        </Typography>
+    </InfoCard>
+    )
+};
+
+const getDisplayForNextOncalls = (oncalls: Oncall[]) => {
```

Comment:
Same here re: plain function vs React.FC.

Reply:
moved to inline